cmd/app/createca.go

```diff
@@ -46,7 +46,7 @@ such as organization, country etc. This can then be used as the root
 certificate authority for an instance of sigstore fulcio`,
 	Run: func(cmd *cobra.Command, args []string) {
 		log.Logger.Info("binding to PKCS11 HSM")
-		p11Ctx, err := crypto11.ConfigureFromFile("config/crypto11.conf")
+		p11Ctx, err := crypto11.ConfigureFromFile(viper.GetString("crypto-config-path"))
```

Comment:
Good idea!

Could we name it `pkcs11-config-path` instead please?

Reply:
sure

Reply:
updated